pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -217,6 +217,11 @@ type ClusterSpec struct {
 	// +optional
 	// +nullable
 	LogCollector LogCollectorSpec `json:"logCollector,omitempty"`
+
+	// specify externalIPs
+	// +optional
+	// +nullalbe
+	ExternalIps []string `json:"externalIPs,omitempty"`
```

Comment:
Let's add this in the `DashboardSpec` instead of the `ClusterSpec`

Reply:
You'll need to run `make codegen` and `make crds` to generate the artifacts needed for this setting.